### PR TITLE
Turnaround improvements

### LINKED ISSUE #995: Remove "landing turn" routine
remove `FIGHTER_INSTANCE_WORK_ID_FLAG_DISABLE_LANDING_TURN` check from `sub_landing_uniq_process_init_main_param`

### PR DESCRIPTION
**Pull request by jobrien97**
- Removes a ""tech"" from vanilla which turned you around *during* your landing animation if you had the stick held backwards right before landing
- Makes regular turnarounds (not smash turns/dashbacks) unable to be buffered, which should prevent any unintentional turnarounds some may have experienced during movement

Resolves #995

### CODE REVIEW

**Comment by Riah321** [comment]
Does this address issue #964?

**Comment by jobrien97** [comment]
> Does this address issue #964?

Maybe, test it out

File: fighters/common/src/general_statuses/mod.rs [hunk 1]
```diff
@@ -141,7 +141,7 @@ fn nro_hook(info: &skyline::nro::NroInfo) {
             sub_transition_group_check_ground_jump_mini_attack,
             sub_transition_group_check_air_escape,
             sub_transition_group_check_ground_escape,
-            sub_transition_group_check_ground_guard
+            sub_transition_group_check_ground
```
Comment (WuBoytH) [inline fighters/common/src/general_statuses/mod.rs:145]: You removed the `sub_transition_group_check_ground_guard` install, was that intentional?

Comment (jobrien97) [reply fighters/common/src/general_statuses/mod.rs:145]: > You removed the `sub_transition_group_check_ground_guard` install, was that intentional?

No it was not…

Comment (WuBoytH) [reply fighters/common/src/general_statuses/mod.rs:145]: lmao gottem

**Review by WuBoytH — APPROVED** [review]
https://www.youtube.com/watch?v=sAXZbfLzJUg

**Pull request merged**